### ISSUE TITLE
Excessive battery usage due to health app logging? (iOS 13.5.1)

### ISSUE THREAD

**Issue by leejo** [DP-3T/dp3t-sdk-ios#174]
Device info: iPhone 11 Pro. iOS 13.5.1

According to the [FAQ](https://www.bag.admin.ch/bag/en/home/krankheiten/ausbrueche-epidemien-pandemien/aktuelle-ausbrueche-epidemien/novel-cov/faq-kontakte-downloads/haeufig-gestellte-fragen.html?faq-url=/en/swisscovid-app-how-it-works/what-are-effects-app-battery-life-0) the impact the application has on battery life should be relatively low.

Having installed the app last week I have noticed my battery requires much more frequent charging. I had a look at the battery statistics this morning:

![IMG_8020](https://user-images.githubusercontent.com/97388/86213601-3d798600-bb7a-11ea-897e-d1773c884ae0.jpg)

Over a 24hr period the level of my battery fell from approx 75% to approx 25%. According to the battery stats 80% of that was the health app logging data from the SwissCovid app. That seems rather excessive.

Is there an issue here, or is this expected behaviour?

**Comment by UBaggeler** [comment]
Exposure Notifications of iOS are optimized by Apple to save as much battery as possible. These reported battery usage numbers are relative to what is running on a phone during a certain time. When nothing else is running, COVID-tracing can take up to 100% of battery (because no other app is using battery). During testing we have observed a low usage of EN compared to when it was disabled.

**Comment by leejo** [comment]
Yep, I understand that the reports are relative, but 80% of an approx 40% reduction in battery charge still seems rather excessive.

Here's yesterday, which is much more as expected - only falling a couple of percent. I think the major difference here is that I finished charging the phone before going home at 5pm, so it will not have been in the presence of any other device after having been fully charged.

![IMG_8023](https://user-images.githubusercontent.com/97388/86322808-eb009e00-bc3b-11ea-9241-2281698b23f1.jpg)

I shall leave my phone on my desk all day today without charging it and see what the reports say tomorrow.

**Comment by leejo** [comment]
Here's yesterday and the last ten days.

These look a lot more reasonable. I think this is still something worth looking into, there was a poll done on the NHS' app that revealed [even a small to moderate drain on battery life would put people off installing it](https://yougov.co.uk/topics/technology/articles-reports/2020/05/14/nhs-covid-19-app-success-hinges-battery-life).

Obviously I will be keeping the app installed, but I think there is a danger if users look at their battery life usage and see figures like this they might be tempted to uninstall the app.

![IMG_8026](https://user-images.githubusercontent.com/97388/86438202-67f84a00-bd06-11ea-8bf9-72bebc31d5ab.PNG)

![IMG_8027](https://user-images.githubusercontent.com/97388/86438209-6af33a80-bd06-11ea-9285-f10703abfc3d.PNG)

**Comment by UBaggeler** [comment]
Can you please re-check with iOS 13.6?

**Comment by leejo** [comment]
> Can you please re-check with iOS 13.6?

Yep. Will let you know in a couple of days.

**Comment by leejo** [comment]
No change as far as I can see.

**Comment by leejo** [comment]
https://youtu.be/cRH-mrxrJHw?t=397
